.pre-commit-config.yaml

```diff
@@ -91,6 +91,7 @@ repos:
         name: Check typing with mypy
         args:
           - --config-file=pyproject.toml
+          - --explicit-package-bases
```

Comment:
can you explain why we need this?

Reply:
This is related to an issue we were having with mypy discovering imports --> https://anta.arista.com/main/contribution/#configure-mypypath

MYPYPATH used to do the trick but not anymore. Didn't investigate much. Here is the error I am getting:
```
Check typing with mypy...................................................Failed
- hook id: mypy
- exit code: 2

/home/cbaillar/git_projects/__init__.py: error: Source file found twice under different module names: "git_projects.anta.anta.logger" and "anta.logger"
/home/cbaillar/git_projects/__init__.py: note: See https://mypy.readthedocs.io/en/stable/running_mypy.html#mapping-file-paths-to-modules for more info
/home/cbaillar/git_projects/__init__.py: note: Common resolutions include: a) adding `__init__.py` somewhere, b) using `--explicit-package-bases` or adjusting MYPYPATH
Found 1 error in 1 file (errors prevented further checking)
```

Can you test in your environment? I remember @titom73 and/or @mtache also had this issue in the past.

Adding this flag fixes it for now:P --> https://mypy.readthedocs.io/en/stable/running_mypy.html#mapping-paths-to-modules

Reply:
TBC - probably not needed